packages/grafana-ui/src/utils/standardEditors.tsx

```diff
@@ -35,23 +37,26 @@ import { UnitValueEditor } from '../components/OptionsUI/units';
 import { DataLinksValueEditor } from '../components/OptionsUI/links';
 import { ColorValueEditor } from '../components/OptionsUI/color';
 import { StatsPickerEditor } from '../components/OptionsUI/stats';
+import { StringWithSuggestionsValueEditor } from '../components/OptionsUI/string';
 
 /**
  * Returns collection of common field config properties definitions
  */
 export const getStandardFieldConfigs = () => {
   const category = ['Standard options'];
-  const displayName: FieldConfigPropertyItem<any, string, StringFieldConfigSettings> = {
+
+  const displayName: FieldConfigPropertyItem<any, string, StringFieldWithSuggestionsConfigSettings> = {
     id: 'displayName',
     path: 'displayName',
     name: 'Display name',
     description: 'Change the field or series name',
-    editor: standardEditorsRegistry.get('text').editor as any,
-    override: standardEditorsRegistry.get('text').editor as any,
+    editor: standardEditorsRegistry.get('textWithSuggestions').editor as any,
+    override: standardEditorsRegistry.get('textWithSuggestions').editor as any,
     process: stringOverrideProcessor,
     settings: {
       placeholder: 'none',
       expandTemplateVars: true,
```

Comment:
can we use `expandTemplateVars` as a key to add the standard variables suggestions to whatever else is added?  or can/should we have another parameter `showTemplateVarSuggestions`

Reply:
Ha, this is a very good point, completely forgot we added this property a long time ago. Since this enforces variables interpolation I think it's a good move to use it for enabling the input with suggestions.